composer.json

```diff
@@ -17,26 +17,26 @@
         "doctrine/doctrine-migrations-bundle": "^3.0",
         "doctrine/orm": "^2.5.11",
         "erusev/parsedown": "^1.6",
-        "sensio/framework-extra-bundle": "^5.1",
+        "sensio/framework-extra-bundle": "^5.6",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.1",
-        "symfony/console": "^5.1",
-        "symfony/dotenv": "^5.1",
-        "symfony/expression-language": "^5.1",
+        "symfony/asset": "5.2.0-beta1",
```

Comment:
use `^5.2.0-beta1` to avoid CI failures on `composer validate --strict`

Reply:
Done. Thanks.